app/assets/config/manifest.js

```diff
@@ -0,0 +1,28 @@
+//= link_tree ../../../vendor/assets/fonts
+//= link_tree ../images
+//= link application.css
+//= link application.js
+//= link vendor.js
+
+//= link editable/clear.png
+//= link bg-modal-about-pf.png
+
+//= link unimported/email.css
+
+// link compute_resource.js
```

Comment:
Probably forgotten file in precompiles, as it doesn't exists, but I've added it to be noticed it's not compiled now, do someone know if it is safe to remove?
(same goes for `reports.js`, `gettext/all.js`)

Reply:
I believe these are all leftovers that were moved to webpack in the past and can be dropped from here.